src/layer/Popup.js

```diff
@@ -237,12 +230,10 @@ L.Popup = L.Layer.extend({
 	getEvents: function () {
 		var events = {
 			zoom: this._updatePosition,
-			viewreset: this._updatePosition
+			viewreset: this._updatePosition,
+			zoomanim: this._animateZoom
 		};
 
-		if (this._zoomAnimated) {

```

Comment:
ok, so if we keep this._zoomAnimated from L.Layer and remove the option
from Popup, we still need to fix the updatePosition test in the case where
zoomAnimated is false. Couldn't figure out the right calculation yet

On Mon, Jul 4, 2016 at 9:23 PM, Yohan Boniface <notifications@github.com>
wrote:

> In src/layer/Popup.js
> <https://github.com/Leaflet/Leaflet/pull/4699#discussion_r69486833>:
>
> >  		};
> >
> > -		if (this._zoomAnimated) {
>
> Note: keep in mind that this._zoomAminated comes from L.Layer.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Leaflet/Leaflet/pull/4699/files/3df8411b7998221d5aa171a0c1343b927924d46b#r69486833>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABLtode2pe0gFPgQYn2iRilqM2_ru3_7ks5qSV2tgaJpZM4JDR9->
> .
>



-- 
Fabien Nicollet
Tel : +33 (0)6 60 54 72 20